conf/global.json

```diff
@@ -30,6 +30,7 @@
     },
     "rules_table": {
       "enabled": false,
+      "refresh_minutes": 10,
```

Comment:
This makes me think this is a refresh of the table itself. But this refers to the rule processor refreshing its in-memory cache, right? If so, maybe `cache_refresh_minutes` would be clearer?

Reply:
I like it! will make the change